CHARTER.md

```diff
@@ -1,107 +1,121 @@
+- c. Participation in a call for consensus is open to anyone in the technical community that contributes text, source code, documentation or other artifacts in the source repository of the WG or any of its Technical Initiatives.
 
-   - ii. Collaborators are Contributors who have earned the ability to modify ("commit") text, source code, documentation or other artifacts in the Technical Initiative’s repository or direct the agenda or working activities of the Technical Initiative; and
+- d. The TSC may create, change, modify, or remove roles or their definitions, so long as the definitions of roles for the WG are publicly available in the WG repository.
 
-   - iii. A Contributor may become a Collaborator by a majority approval of the existing Collaborators. A Collaborator may be removed by a majority approval of the other existing Collaborators.
+- e. The TSC will be responsible for all aspects of oversight relating to the WG, which may include:
 
-   - iv. Maintainers are the initial Collaborators defined at the creation of the Technical Initiative. The Maintainers will determine the process for selecting future Maintainers. A Maintainer may be removed by two-thirds approval of the other existing Maintainers, or a majority of the other existing Collaborators.
+  - i. coordinating the direction of the WG;
+  - ii. approving, organizing or removing activities;
+  - iii. establish community norms, workflows, processes, release requirements, and templates for the operation of the WG;
+  - iv. appointing representatives to work with other open source or open standards communities;
+  - v. approving and implementing policies and processes for contributing (to be published in the WG repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
+  - vi. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the WG; and
+  - vii. coordinating any communications regarding the WG.
 
-- d. Participation in the Technical Initiative through becoming a Contributor, Collaborator, or Maintainer is open to anyone, whether a OpenSSF member or not, so long as they abide by the terms of this Technical Charter. 
+- f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC. The TSC Chair will serve as the WG’s voting representative on the TAC. In the event that a proxy is requested by the TSC Chair, it may be any other TSC member so designated by a silent consensus of the TSC.
 
-- e. The TSC may create, change, modify, or remove roles or their definitions, so long as the definitions of roles for the Technical Initiative are publicly available in the Technical Initiative repository.
+- g. The current TSC Chair is:
```

Comment:
I'd argue that the current chair (or any other holder of a formalized role) should not be listed in the charter. The charter should define (at least some of) the roles, and the roster should be kept in a separate file. This makes it so that changes to the charter are only for issues with the charter itself.

Reply:
That's fine with me

For context, I was inclined this direction in a desire to make this a single source of truth for the WG governance. Especially since most of the work is done outside of this repo and the WG is simply a coordination body, it seems like we should push the charter into line-of-sight however we can.

Reply:
I can see that reasoning. I don't think it matters a ton either way, so I'm content with leaving this as-is. We can always pull it out to a separate file later if needed.

Reply:
That would be my preference, if it's all the same to others